docs/frameworks/laravel.md

```diff
@@ -67,46 +67,34 @@ Follow [the deployment guide](/docs/deploy.md#deploying-for-production) for more
 
 In case your application is showing a blank page after being deployed, [have a look at the logs](../environment/logs.md).
 
-## Trusted proxies
-
-Because Laravel is executed through API Gateway, the `Host` header is set to the API Gateway host name. Helper functions such as `redirect()` will use this incorrect domain name. The correct domain name is set on the `X-Forwarded-Host` header.
-
-To get Laravel to use `X-Forwarded-Host` instead, edit the `App\Http\Middleware\TrustProxies` middleware and set `$proxies` to the `*` wildcard:
-
-```php
-class TrustProxies extends Middleware
-{
-    // ...
-    protected $proxies = '*';
-```
```

Comment:
@tillkruss I removed this section from the docs as the new bridge takes care of this, right?

Reply:
I think so. I'll test this all once Bref v2 is out and I run into issues in production 🫠

Reply:
Looks like it: https://github.com/cachewerk/bref-laravel-bridge/blob/0193eddb81819764553ed115b69e3ec65d43c058/src/BrefServiceProvider.php#L45